cms/envs/production.py

```diff
@@ -585,6 +585,10 @@ def get_env_setting(setting):
 
 ######################## CELERY ROTUING ########################
 
+# Celery beat configuration
+
+CELERYBEAT_SCHEDULER = ENV_TOKENS.get('CELERYBEAT_SCHEDULER', CELERYBEAT_SCHEDULER)
```

Comment:
This line is not necessary, all keys in the config file except for a set of exceptions get loaded into the module.  See https://github.com/edx/edx-platform/blob/5b7edd472d92f6cb1a45b91827c5dd98e2092d27/cms/envs/production.py#L56-L81